google-charts/README.md

```diff
@@ -0,0 +1,12 @@
+# Google charts
+
+This Dataiku DSS plugin helps you create, without code, different type of charts.
+
+It contains several webapps for 
```

Comment:
Note: can be released as is (webapps) but @FChataigner is creating proper charts plugins (visible from the charts tab in DSS).
The transition is easy to do because the charts plugins are technically webapps, just not shown as such to end users.

Reply:
Also, we are working on a time-series chart (5.2?) that may have a waterfall option (maybe, see with @jereze )